prow/plugins/label/label.go

```diff
@@ -32,7 +32,7 @@ const pluginName = "label"
 var (
 	labelRegex              = regexp.MustCompile(`(?m)^/(area|priority|kind|sig)\s*(.*)$`)
 	removeLabelRegex        = regexp.MustCompile(`(?m)^/remove-(area|priority|kind|sig)\s*(.*)$`)
-	nonExistentLabelOnIssue = "Those labels are not set on the issue: `%v`"
+	nonExistentLabelOnIssue = "Those labels are not set on issue #%d: `%v`"
```

Comment:
>the comment is created on the issue itself.

yes but I was thinking to print the issue number in the comment to help debug in case there is similar issue as #4921 :) I'm not strongly on it though.

Reply:
If it just for debugging from logs then that should be recorded in logs not in the comment contents itself. That info is already logged by the github client itself see [here](https://github.com/kubernetes/test-infra/blob/master/prow/github/client.go#L275).

I think including the issue number in the comment could be confusing to users. It suggests that there is a problem with labels on another issue until the user realizes that the number is their own issue number. Including it as a sanity check seems a little gross to me.

Reply:
Agree :)
/close